src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/WorkingDaysService.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.springframework.stereotype.Service
+import java.time.DayOfWeek
+import java.time.LocalDate
+
+@Service
+class WorkingDaysService(private val bankHolidayService: BankHolidayService) {
+
+  fun addWorkingDays(date: LocalDate, workingDays: Int): LocalDate {
+    val bankHolidays = bankHolidayService.getBankHolidaysForEnglandAndWales()
+    var adjustedDate = date
+    for (i in 0 until workingDays) {
```

Comment:
you can probably remove this unused `i` variable somehow.

We calculate something very similar to this in the ReleaseDateService.

Reply:
I'll merge this now but we could look at merging the two implementations

Reply:
Removed the `I` in place of a repeat. Re release date service, I'll refactor this in next PR to use this instead of the implementations we have in the `ReleaseDateService`